python/src/trezorlib/ethereum.py

```diff
@@ -26,6 +30,18 @@
     from .protobuf import MessageType
 
 
+# TODO: change once we know the urls
+DEFS_BASE_URL = "https://firmware.corp.sldev.cz/eth-definitions/"
+DEFS_URL_LOOKUP_TEMPLATE = DEFS_BASE_URL + "{lookup_type}/{id}/{name}"
+
+DEFS_ZIP_FILENAME = "definitions-latest.zip"
+DEFS_ZIP_TOPLEVEL_DIR = pathlib.PurePath("definitions-latest")
+DEFS_NETWORK_BY_CHAINID_LOOKUP_TYPE = "by_chain_id"
+DEFS_NETWORK_BY_SLIP44_LOOKUP_TYPE = "by_slip44"
+DEFS_NETWORK_URI_NAME = "network.dat"
+DEFS_TOKEN_URI_NAME = "token_{hex_address}.dat"
```

Comment:
This could have the suffix `_TEMPLATE` to signify it expects data to be injected

Reply:
code was removed